index.js

```diff
@@ -188,10 +188,11 @@ var bundleWithWebpack = function(fn, fnModuleId) {
   var modulesString;
 
   if (devMode) {
-    fnModuleId = '"' + fnModuleId + '"';
+    // must escape quotes to support webpack loader options
+    fnModuleId = '"' + fnModuleId.replace(/"/g, '\\\"') + '"';
```

Comment:
@mjneil  why not ?

```js
fnModuleId = JSON.stringify(fnModuleId);
```

looks much cleaner and does same thing

Reply:
Ah good call, that is nicer